app/src/components/Card.css

```diff
@@ -0,0 +1,14 @@
+.card {
+  width: 376px;
+  height: 450px;
+  flex-direction: column;
+  gap: 40px;
+  margin: 62px 50px;
+  border-radius: 10px;
+  border: 1px solid var(--second-color);
```

Comment:
nice. i love you already use `vars`. we gonna need that when we implement night/light theme for this app in the future